docs/weave/welcome.md

```diff
@@ -101,19 +101,19 @@ The application is fed `genesis.json` the first time it starts up via the `InitC
 
 ### Setting the Validators
 
-Since Tendermint uses a traditional BFT algorithm to reach consensus on blocks, signatures from specified validator keys replace hashes used to mine blocks in typical PoW chains. This also means that the selection of validators is an extremely important part of the blockchain security. [Read More](configuration/validators.html)
+Since Tendermint uses a traditional BFT algorithm to reach consensus on blocks, signatures from specified validator keys replace hashes used to mine blocks in typical PoW chains. This also means that the selection of validators is an extremely important part of blockchain security. [Read More](configuration/validators.html)
 
 ## Building your own Application
 
-Before we get into the strucutre of the application, there are a few design principles for weave (but also tendermint apps in general) that we must keep in mind.
+Before we get into the structure of the application, there are a few design principles for weave (but also tendermint apps in general) that we must keep in mind.
 
 ### Determinism
 
 The big key to blockchain development is determinism. Two binaries with the same state must **ALWAYS** produce the same result when passed a given transaction. [Read More](design/overview.html#determinism)
 
 ### Abstract Block Chain Interface (ABCI)
 
-To understand this design, you should first understand what an ABCI application is and how that level blockchain abstraction works. ABCI is the interface between the tendermint daemon and the state machine that processes the transactions, something akin to wsgi as the interface between apache/nginx and a django application. [Read More](design/overview.html#abci)
+To understand this design, you should first understand what an ABCI application is and how that abstraction works. ABCI is the interface between the tendermint daemon and the state machine that processes the transactions, something akin to wsgi as the interface between apache/nginx and a django application. [Read More](design/overview.html#abci)
```

Comment:
Please check capitalisation of:
tendermint
daemon
wsgi
apache
nginx
django